### PR TITLE
🐛 amp-next-page: Fix duplicate custom separators for child pages with amp-next-page

### PR DESCRIPTION
**Pull request by peterjosling**
Child pages with `amp-next-page` tags of their own will display double custom separators.

`display: none` is added to the custom separator when the component registers, which it never does on child pages. This change removes the original separator from the DOM entirely when the component is constructed.

### CODE REVIEW

**Comment by peterjosling** [comment]
@aghassemi

File: extensions/amp-next-page/0.1/amp-next-page.js [hunk 1]
```diff
@@ -41,6 +42,19 @@ export class AmpNextPage extends AMP.BaseElement {
 
     /** @private {!./next-page-service.NextPageService} */
     this.service_ = getService(this.win, SERVICE_ID);
+
+    /** @private {?Element} */
+    this.separator_ = null;
+
+    const separatorElements = childElementsByAttr(element, 'separator');
```
[the remark this answers is not in the document]
Comment (peterjosling) [reply extensions/amp-next-page/0.1/amp-next-page.js]: Done

**Comment by aghassemi** [comment]
@peterjosling This is approved but there is a `amp-next-page` unit test failure which is not a flake.

**Comment by peterjosling** [comment]
The flake should have been fixed in https://github.com/ampproject/amphtml/pull/16624

**Comment by peterjosling** [comment]
Oops, misread your comment.

Issue was from the merge, fixed manually.

**Pull request merged**